kubernetes/che-plugin-registry/values.yaml

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2012-2017 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+chePluginRegistryImage: eclipse/che-plugin-registry
+chePluginRegistryImagePullPolicy: Always
+chePluginRegistryMemoryLimit: 256Mi
+
+global:
+  ingressDomain: 192.168.99.100.nip.io
```

Comment:
IP is hardcoded ?

(Missing also a newline)

Reply:
Added the new line. For the ingressDomain default value I am using the hardcoded IP we use in Che Server Helm chart. We still haven't found a better solution.